### PR TITLE
huge cleanup of tigerAddressImport.py

### PR DESCRIPTION
**Pull request by mtmail**
- remove all logic that created *.osm files (were not used)
- rename variables to make it more clear that SQL lines are created
- move county configuration to separate *.json file. No change to that data. I will file another PR where the county data gets updated slightly
- remove many unused variables, hash keys, imported libraries
- moved SQL quoting into a method

The script output is the same. It's sorted differently though.

```
utils/tigerAddressImport.py tl_2017_05023_edges.shp new_05023.sql
sort new_05023.sql > /tmp/05023.new.sorted.sql
sort 2017tiger/05023.sql > /tmp/05023.old.sorted.sql # file from https://github.com/openstreetmap/Nominatim/pull/819#issuecomment-338397402
diff -i /tmp/05023.new.sorted.sql /tmp/05023.old.sorted.sql
# no output
```

### CODE REVIEW

**Comment by lonvia** [comment]
Cool. I'll try it on the complete set before merging.

File: utils/imports.php [hunk 1]
```diff
@@ -28,20 +28,22 @@
...
         $sCountyID = $aMatch[1];
+
         echo "Processing ".$sCountyID."...\n";
         $sUnzipCmd = "unzip -d $sTempDir $sImportFile";
         exec($sUnzipCmd);
-        $sShapeFile = $sTempDir.'/'.basename($sImportFile, '.zip').'.shp';
-        if (!file_exists($sShapeFile)) {
+
+        $sShapeFilename = $sTempDir.'/'.basename($sImportFile, '.zip').'.shp';
+        $sSqlFilename = CONST_Tiger_Data_Path.'/'.$sCountyID.'.sql')
```
Comment (lonvia) [inline utils/imports.php]: This looks wrong.

**Comment by mtmail** [comment]
corrected the `utils/imports.php`

**Comment by lonvia** [comment]
I see the following now:

```
Traceback (most recent call last):
  File "/home/lonvia/Nominatim/utils/tigerAddressImport.py", line 619, in <module>
    shape_to_sql(shp_filename, sql_filename)
  File "/home/lonvia/Nominatim/utils/tigerAddressImport.py", line 605, in shape_to_sql
    sql_lines = addressways(waylist, nodelist, i)
  File "/home/lonvia/Nominatim/utils/tigerAddressImport.py", line 454, in addressways
    ( ",".join(rlinestring), sql_quote(rfromadd), sql_quote(rtoadd), sql_quote(interpolationtype), sql_quote(name), sql_quote(county), sql_quote(zipr) ) )
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 134: ordinal not in range(128)
Failed parse (/home/lonvia/tiger-src/TIGER2017/EDGES/tl_2017_04013_edges.zip)
```

**Comment by mtmail** [comment]
This name is the cause `E Cír S Hernandez B`

**Comment by lonvia** [comment]
So converting the county name from unicode to str did the trick. Afterwards all files pass and are the same as the one from the previous version (when sorted and case is ignored).

Merged manually together with a fix for the county string type.

**Pull request closed without merging**